server/public/javascript/sanitize.js

```diff
@@ -0,0 +1,15 @@
+const sanitizeEmails = (customers) => {
+
+  for (const customer of customers) {
+    customer.email = customer.email.replace(/[&\/\\#,+()$~%'":*?<>{}\s]/g, '').replace('..', '')
```

Comment:
I would recommend doing this a bit different.[ What about sequelize Hooks?](https://sequelize.org/master/manual/hooks.html) This way every time we try to create a new customer, we are going to make sure the email is going to be sanitized.